docs/index.js.html

```diff
@@ -160,27 +157,37 @@ <h1 class="page-title">Source: index.js</h1>
   if (hasBrowserSupport) {
     return;
   }
-  insertSpanningStyles(element);
-  feature.addEventListener("change", () => insertSpanningStyles(element));
+  insertViewportSegmentsStyles(element);
+  feature.addEventListener("change", () => insertViewportSegmentsStyles(element));
 }
 
-function insertSpanningStyles(element) {
+function insertViewportSegmentsStyles(element) {
   let options = feature;
+  let viewportSegmentsDefinitions;
+  if (element)
+  viewportSegmentsDefinitions = viewportSegments[element.nodeName.toLowerCase()];
+  else
+  viewportSegmentsDefinitions = viewportSegments;
 
-  let spanningCSSText = element ?
-    spanning[element.nodeName.toLowerCase()][options.screenSpanning] :
-    spanning[options.screenSpanning];
+  let viewportSegmentCSSText = viewportSegmentsDefinitions[options.verticalViewportSegments] ?
+    viewportSegmentsDefinitions[options.verticalViewportSegments][options.horizontalViewportSegments] ?
+      viewportSegmentsDefinitions[options.verticalViewportSegments][options.horizontalViewportSegments] : null : null;
```

Comment:
: null : null, maybe add () around them to make it more readable, or split it up?